Sources/SwiftSyntax/gyb_generated/syntax_nodes/SyntaxNodes.swift

```diff
@@ -6901,6 +6901,947 @@ extension ObjCSelectorPieceSyntax: CustomReflectable {
   }
 }
 
+// MARK: - DifferentiableAttributeArgumentsSyntax
+
+/// 
+/// The arguments for the `@differentiable` attribute: an optional          differentiation parameter list and associated functions.
```

Comment:
Note: this description comment is suboptimal and will be fixed in a follow-up.
Tracked here: https://github.com/apple/swift/pull/27506#discussion_r345306776